django_switchuser/state.py

```diff
@@ -11,6 +11,8 @@ def __init__(self, request):
         self._reset()
 
     def _reset(self):
+        User = get_user_model()
```

Comment:
@wolever https://docs.djangoproject.com/en/1.9/topics/auth/customizing/#referencing-the-user-model